README.md

```diff
@@ -113,7 +112,8 @@ spec:
 
 ### 3.8.0
 
-+ Added memory volume metric for cache monitoring
++ Added memory volume metric for cache monitoring such as actual buffer count and size
```

Comment:
I think we should add the actual names for metrics and describe them in the README file. Otherwise, we will have to go to the code to get the metrics' names.
Also, it would be nice to have the description for labels that are used in these metrics